lib/services/android-project-service.ts

```diff
@@ -298,6 +298,10 @@ export class AndroidProjectService extends projectServiceBaseLib.PlatformProject
 		return this.$fs.exists(path.join(this.platformData.appDestinationDirectoryPath, constants.APP_FOLDER_NAME));
 	}
 
+	public isPlatformBuilt(projectRoot: string): IFuture<boolean> {
+		return this.$fs.exists(path.join(projectRoot, "build", "outputs", "apk"));
```

Comment:
I don't believe livesync should depend on the `platforms/android/build` folder at all. Folders inside build are temporary, they are being constantly changed and we can't be sure if this won't have unexpected behavior depending on the user actions.